packages/nextjs/components/impact-vector/ImpactVectorGraph.tsx

```diff
@@ -121,7 +121,13 @@ export default function ImpactVectorGraph({ data }: { data: DataSet[] }) {
             tickMargin={10}
             scale={isLogarithmic ? logScale : "linear"}
             domain={["auto", "auto"]}
+            tickFormatter={abbreviateNumber}
```

Comment:
Added this number abbreviation to reduce the overall width of the graph to make room for the new label. It's a little unconventional to do this for numbers above 1 million (`3000k` feels a bit strange), so I can try to find a different way to make space for the `OP Allocation` label if y'all want